script/add-asset-hashes.js

```diff
@@ -56,7 +56,9 @@ function addAssetHashes() {
     ].forEach(unhashedName => {
       const hashedName = manifest[unhashedName];
 
-      files[`generated/${unhashedName}`] = files[hashedName.substr(1)]; // eslint-disable-line no-param-reassign
+      if (hashedName) {
+        files[`generated/${unhashedName}`] = files[hashedName.substr(1)]; // eslint-disable-line no-param-reassign
+      }
```

Comment:
```
[
       'proxy-rewrite.entry.js',
       'styleConsolidated.css',
       'static-pages.css',
       'vendor.entry.js',
       'polyfills.entry.js',
    ]
```
Some context for these; apologies if you already know this - These bundles are all required for the Teamsite injection work (applying our header/footer to VA sites outside of our source code), but there was a problem encountered with the hashed bundle names (cache-busting in higher environments). So, this little piece reverses that hashing. It should probably be assigned to a variable for clarity.

Down the road I think I would wonder at what point is `hashedName` not there. Since this seems to be a situation with the `--entry` flag, I wonder if it would be more clear if we used that argument to apply a filter to that array?

Reply:
I'm not sure I follow. Are you suggesting rather than checking if `hashedName` is truthy, check instead for the presence `--entry` flag?

Reply:
I think that if there is an `--entry` argument that isn't `proxy-rewrite`, then we can skip that entire `forEach` loop. 

But also a comment there would probably suffice 😆